src/System.Memory/tests/Performance/Perf.Span.BinarySearch.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Memory.Tests
+{
+    public class Perf_Span_BinarySearch
+    {
+        // Existing.
+        // List:
+        // https://github.com/dotnet/corefx/blob/157fff35c2427dd0ce5e79557d506e8e7947c8d4/src/System.Collections/tests/Performance/Perf.List.cs#L550
+        // https://github.com/dotnet/corefx/blob/157fff35c2427dd0ce5e79557d506e8e7947c8d4/src/System.Collections/tests/Performance/Perf.List.cs#L577
+        // TODO: What tests do we need here? What types (short, int, string ...)? Range? Scenarios?
```

Comment:
Similar to IndexOf tests (https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Performance/Perf.Span.IndexOf.cs), I would say String and Int32 should be sufficient.

For range, [1, 10, 100, 1000].
For scenarios, consider the location of the value being searched for in the span? Maybe first index, middle index, last index? And lastly, if value isn't in the span at all.